src/apps/frontend/services/auth.service.ts

```diff
@@ -22,4 +22,24 @@ export default class AuthService extends APIService {
     username,
     password,
   });
+
+  sendOTP = async (
+    phoneNumber: PhoneNumber,
+  ): Promise<ApiResponse<void>> => this.apiClient.post('/accounts', {
```

Comment:
Please check the below comment over ln. 38. The same is true here, it will return the json object instead of void. Correct me if I'm wrong ideally it should look similar to
  ```
sendOTP = async (
    phoneNumber: PhoneNumber,
  ): Promise<ApiResponse<void>> => {
    await this.apiClient.post('/accounts', {
      phoneNumber,
    });

    return new ApiResponse();
  };
```

Reply:
https://github.com/jalantechnologies/rflask-boilerplate/pull/52#discussion_r1744930394